### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by KonradStaniec**

### CODE REVIEW

File: src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala [hunk 1]
```diff
@@ -44,7 +44,7 @@ class PeerDiscoveryManagerSpec extends FlatSpec with Matchers with MockFactory w
 
     val expectedFindNodeResponse = SendMessage(FindNode(ByteString(nodeStatus.nodeId), expectedTime), remoteUdpAddress)
 
-    Thread.sleep(500)
+    Thread.sleep(1500)
```
Comment (AlanVerbner) [inline src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala]: Why this change was introduced in a `README` update PR?

File: build.sbt [hunk 2]
```diff
@@ -2,7 +2,7 @@ enablePlugins(JDKPackagerPlugin, JavaAppPackaging, SolidityPlugin)
 
 val commonSettings = Seq(
   name := "mantis",
-  version := "1.0-cli-rc1",
+  version := "1.0-cli",
```
Comment (AlanVerbner) [inline build.sbt]: Do we need to update somewhere in `Daedalus` where Mantis is launched as well?

File: src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala (hunk 1 above)
Comment (mcsherrylabs) [reply src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala]: Is this still necessary?

File: build.sbt (hunk 2 above)
Comment (KonradStaniec) [reply build.sbt]: Actually this branch is totally separated from branch integrating with `Daedalus`  so no changes necessary

File: src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala (hunk 1 above)
Comment (KonradStaniec) [reply src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala]: Cli release is a totally different branch from daeadalus release branch. This timeout increase was introduced already in `phase/daeadlus` release and in `phase/release1_1` so to avoid spurious tests fails it made sense to introduce it here.
